src/adapter/src/coord/sequencer/linked_cluster.rs

```diff
@@ -64,7 +68,7 @@ impl Coordinator {
                 variant: ClusterVariant::Unmanaged,
             },
         });
-        self.create_linked_cluster_replica_op(id, size, ops, *session.current_role_id())
+        self.create_linked_cluster_replica_op(id, size, disk, ops, *session.current_role_id())
```

Comment:
one oddity here is that if `disk_cluster_replicas_default=true` then a sink's linked cluster could get assigned disk. but I don't think the intent is to set `disk_cluster_replicas_default=true` in prod, mostly for testing, so I'm not sure if this is a concern

Reply:
Yeah, that seems fine to me. Seems right, to be honest, to have linked clusters follow the default for `disk_cluster_replicas_default`.

Reply:
yeah, sounds fine to me as well!